store/opentracing_layer.go

```diff
@@ -5431,6 +5449,42 @@ func (s *OpenTracingLayerRoleStore) Save(role *model.Role) (*model.Role, *model.
 	return resultVar0, resultVar1
 }
 
+func (s *OpenTracingLayerSchemeStore) CountByScope(scope string) (int64, *model.AppError) {
```

Comment:
No need to add these changes, I've made a PR for it [here](https://github.com/mattermost/mattermost-server/pull/14261)

Reply:
Great! Updated the PR to remove everything but the check itself then :)